automated install/basic-install.sh

```diff
@@ -2304,6 +2304,25 @@ main() {
     fi
   fi
 
+  # resolved and dnsmasq can't share port 53. 
+  # resolved needs to remain in place for installer to download needed files
+  # so this change needs to be made after installation is complete, but before resarting dnsmasq/ftl
```

Comment:
How about moving this into its own function and calling this inside `start_service pihole-FTL`?

Reply:
I was wondering about that. Could it be splitting it in two? One called early on to check for the presence of enabled resolved and set a flag there. Then just check that flag in the new start_servicepihole-FTL. Rather than having it tested and applied in the same section? It would allow more granularity in the test if it is needed without cluttering up the body of the script, and might make it cleaner to adjust things should any distros require any special tweaks.